packages/appsync-modelgen-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -359,6 +463,40 @@ export class AppSyncSwiftVisitor<
     return result.join('\n\n');
   }
 
+  protected generateModelPathExtensions(model: CodeGenModel): string {
+    const modelPathExtension = new SwiftDeclarationBlock()
+      .asKind('extension')
+      .withName(`ModelPath where ModelType == ${this.getModelName(model)}`);
```

Comment:
Correct me if I am wrong, the name of extension is just `ModelPath` and the rest is like additional conditions. If this is true, something like
```
.withName('ModelPath')
.withCondition(`ModelType == ${this.getModelName(model)}`);
```
looks more intuitive.

Reply:
Yes that's right, that looks good, updated